docs/contributing.rst

```diff
@@ -149,7 +148,7 @@ and invoke::
 Some tests require optional dependencies to be installed, otherwise
 the tests will be skipped. To install all optional dependencies, run::
 
-    $ pip install -r requirements_dev_optional.txt
+    $ pip install pytest-doctestplus
```

Comment:
I didn't see `pytest-doctestplus` listed anywhere in the optional dependencies, and I don't know what else may have been in `requirements_dev_optional`.

Reply:
I'd be pro putting this in the test dependencies section, and then not having any optional dependencies for testing? That way it's a bit simpler.